localenv/cloud-nine-wallet/docker-compose.yml

```diff
@@ -67,12 +67,10 @@ services:
       TIGERBEETLE_REPLICA_ADDRESSES: ${TIGERBEETLE_REPLICA_ADDRESSES-''}
       AUTH_SERVER_GRANT_URL: ${CLOUD_NINE_AUTH_SERVER_DOMAIN:-http://cloud-nine-wallet-auth:3006}
       AUTH_SERVER_INTROSPECTION_URL: http://cloud-nine-wallet-auth:3007
-      AUTH_ADMIN_API_URL: 'http://cloud-nine-wallet-auth:3003/graphql'
-      AUTH_ADMIN_API_SECRET: 'rPoZpe9tVyBNCigm05QDco7WLcYa0xMao7lO5KG1XG4='
       AUTH_SERVICE_API_URL: 'http://cloud-nine-wallet-auth:3011'
       ILP_ADDRESS: ${ILP_ADDRESS:-test.cloud-nine-wallet}
       STREAM_SECRET: BjPXtnd00G2mRQwP/8ZpwyZASOch5sUXT5o0iR5b5wU=
-      API_SECRET: iyIgCprjb9uL8wFckR+pLEkJWMB7FJhgkvqhTQR/964=
```

Comment:
rename `API_SECRET` in `documentation`?

- rafiki/packages/documentation/src/partials/backend-variables.mdx
- rafiki/packages/documentation/src/partials/backend-variables-docker-compose.mdx

Reply:
> rename API_SECRET in documentation?

I will do that separately in the docs versioning PR